controllers/overlaynetwork_controller.go

```diff
@@ -84,11 +88,21 @@ func (r *OverlayNetworkReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
-func renderNetAttachDef(overlayNet *selfservicev1.OverlayNetwork) *netv1.NetworkAttachmentDefinition {
+func renderNetAttachDef(overlayNet *selfservicev1.OverlayNetwork) (*netv1.NetworkAttachmentDefinition, error) {
 	const netAttachDefKind = "NetworkAttachmentDefinition"
 	const netAttachDefAPIVer = "v1"
-	// TODO: set config
-	netConf := ""
+
+	cniNetConf := map[string]interface{}{
+		"type":             "ovn-k8s-cni-overlay",
+		"name":             overlayNet.GetName(),
```

Comment:
Please rebase on top of the CRD PR and use the correct field.

Reply:
Done